src/components/advanced-search/advanced-search-table.vue

```diff
@@ -177,4 +179,28 @@ export default class AdvancedSearchTable extends Mixins(DateMixin) {
   color: $text;
   line-height: 1.375rem;
 }
+
+// Override the default scroll bar to meet design requirements
+/* width */
+::v-deep ::-webkit-scrollbar {
+  width: 10px;
+}
+
+/* Track */
+::v-deep ::-webkit-scrollbar-track {
+  margin-top: 47px;
+  background: rgb(250,250,250);
+}
+
+/* Handle */
+::v-deep ::-webkit-scrollbar-thumb {
+  background: rgb(193,193,193);
+  border-radius: 10px;
+  height: 2rem;
+}
+
+/* Handle on hover */
+::v-deep ::-webkit-scrollbar-thumb:hover {
+  background: $gray6;
+}
```

Comment:
I'm not sure. Is there a FF alternative? 

I would have to check, but i'm assuming it would fallback to default in FF, which wouldn't be the worst thing.

Reply:
> Those pseudo elements don't work for FF. Should we support FF?

Yes, we support "all major browsers".

Reply:
> I'm not sure. Is there a FF alternative?

The scrollbar customization is more limited but they are following the standard opposed to webkit browsers (most of them 🤔 )
https://developer.mozilla.org/en-US/docs/Web/CSS/scrollbar-width

Reply:
I'm not sure what to action here.
In FireFox this will fall back to default scrollbar values, yes. 

This difference is the small margin at the top of the scrollbar (ie so the scroll thumb doesn't start on the header), which was the design request for this customization.

If you think this is important i can start looking to this customization that is exclusive to this browser (FF)?

Reply:
The question is just a heads up for the difference in FF. If you think it's ok for FF, that's fine. I'm not sure if UX Assurance checks it for every major browser. I can't "Resolve conversations" here, so consider it resolved and thanks for checking!

Reply:
Definitely, i appreciate your opinion here! 
I will look into FireFox a bit here this evening and if i can come up with something that satisfies the browser, will 100% implement it :) 

That being said, if there was no default styling/behaviour it could be an issue, but luckily in this case the default is universal, so it's impact is a little smaller!

Reply:
After a little bit of research into FF, it appears the customization is unfortunately limited to bar width and colour :(